app/src/test/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailViewModelTest.kt

```diff
@@ -139,17 +140,7 @@ class TaskDetailViewModelTest {
     }
 
     @Test
-    fun clickOnEditTask_SetsEvent() {
```

Comment:
Just checking this test isn't needed anymore? I guess it goes with the edit task todo above?

Reply:
Not needed, as this is a UI test for the screen, we shouldn't check for their transitions. That's something we're covering in the end2end tests of AppNavigation and Activity tests